dbt_subprojects/hourly_spellbook/models/_sector/perpetual/projects/pika/optimism/pika_v4_optimism_perpetual_trades.sql

```diff
@@ -0,0 +1,166 @@
+{{ config(
+    schema = 'pika_v4_optimism',
+    alias = 'perpetual_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
```

Comment:
We may simplify the `unique_key` to:
```suggestion
    unique_key = ['tx_hash', 'evt_index'],
```